aries-site/src/layouts/main/Layout.js

```diff
@@ -143,6 +215,42 @@ export const Layout = ({
                 </Box>
               </Main>
             </MainContentWrapper>
+            <FeedbackButton
+              margin={{ vertical: 'large', horizontal: 'medium' }}
+              elevation="large"
+              onClick={onOpen}
+              color="purple!"
+              label="Feedback"
+              primary
+              a11yTitle={`This button launches a modal to give feedback.`}
+            />
+            <Feedback
+              show={open}
+              messages={{
+                submit: 'Submit feedback',
+                cancel: 'No thanks',
+                successful: 'Thank you!',
+              }}
+              modal
+              onClose={onClose}
+              title="We’d love your feedback"
+              onChange={nextValue => setValue(nextValue)}
+              onSubmit={onSubmit}
+            >
+              <Question
+                label="Was this page helpful?"
+                kind="thumbs"
+                name="like-rating"
+              />
+              <Question
+                name="text-area"
+                kind="textArea"
+                label="Any additional comments?"
+                formProps={{
+                  help: `Here's your chance to tell us your thoughts about this page.`,
+                }}
+              />
```

Comment:
Rather than hardcode it here, I was wondering if the names and labels would instead come from the GET API above. It has a labels and displayedFields section that seem to indicate that plus a displayedValues section that has some hint on values it's expecting for the thumbs and such. If you at least get the `names` from the GET then you can more cleanly map the POST fields rather than hardcode some mapping.

Reply:
decided since were not using the api to build out components we are just going to hard code the values for now we can discuss later